### ISSUE TITLE
Idea: Railgun lance/Detris?

### ISSUE THREAD

**Issue by HTMLPhoton** [Arch-Shaman/ZK-Futurewars-Mod#321]
Railguns have been thrown around for a long time. a Piercing hitscan weapon that combines the properties of the ulti disintergrater and that of a laser.

This is now possible thanks to a lot of help from Sprung!

Unsure of what units should get it, replacing the lance tachyon accelerator and detri's gigawatt laser is what came to my mind at first. though unsure if that's actually a good / fun idea.

**Comment by Arch-Shaman** [comment]
Revert exists for a reason. 
I'm working on duck at the moment. Trying out a light skirmisher idea. Let's try it out (detri could be a super dgun or something lol).

There's also blastwaves now if you want to get crazy on that.

**Comment by HTMLPhoton** [comment]
Yup, saw your commits. Now that I think of it the blastwave system might be the missing piece in something I want to do for koda!

Hope you get well soon!

**Comment by Arch-Shaman** [comment]
Kiitos. :) I got it partially done. Archer is kinda fun. The thought I had was having:
- buoy as an assault.
- Grizzly gets demistrider status (making it do something cool)
- Scallop being more rioty (this would nearly complete the raider-riot-skirm circle for amph)
- add some raider (grepe could be fun)

**Comment by Arch-Shaman** [comment]
Grizzly is actually a nice target for a railgun. After 3b834f4 amph now has a light skirmisher/artillery unit in the form of duck. It still needs a bit of work, but it's at least presentable now. 

I want to do some play testing today or tomorrow with terve886.
